src/tacacs/nss/patch/0001-Modify-user-map-profile.patch

```diff
@@ -446,8 +446,12 @@ index 79e62b9..ecfa0b0 100644
 +            }
 +            else if(!strncmp(token, "timeout=", 8)) {
 +                tac_srv[tac_srv_no].timeout = (int)strtoul(token + 8, NULL, 0);
-+                if(tac_srv[tac_srv_no].timeout < 0)
-+                    tac_srv[tac_srv_no].timeout = 0;
++                if(tac_srv[tac_srv_no].timeout < 0) {
++                   tac_srv[tac_srv_no].timeout = 0;
++                }
++                else {
```

Comment:
>else [](http://example.com/codeflow?start=17&length=4)

if tac_srv[tac_srv_no].timeout == 0, do you want to tac_readtimeout_enable or not?

Reply:
Fixed, when config file has a negative config value, timeout will set to 0 and tac_readtimeout_enable will disabled.